WooCommerce/Classes/POS/Presentation/Payments Onboarding/PointOfSaleCardPresentPaymentOnboardingViewModel.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+final class PointOfSaleCardPresentPaymentOnboardingViewModel: ObservableObject {
+    let onboardingViewModel: CardPresentPaymentsOnboardingViewModel
```

Comment:
❓ Question: Would it be the same to have internal `let` that `private(set) var` here? I'm assuming is exactly the same because we're not modifying it after initialization?

Reply:
`private(set) var` still allows the class to change its value internally, while `let` disallows it. Since the onboarding view model `CardPresentPaymentsOnboardingViewModel ` source of truth comes from above (TotalsVM for now), it's not expected to be mutated internally.

Reply:
That makes sense, thanks for the clarification